### ISSUE TITLE
thanos rule graph UI is missing

### ISSUE THREAD

**Issue by Wing924** [thanos-io/thanos#1503]
**Thanos, Prometheus and Golang version used**
```
thanos, version 0.7.0 (branch: HEAD, revision: c6eaf68bec32aefd280318eaef6121d6ddb20d10)
  build user:       root@aa355daaf987
  build date:       20190902-15:31:30
  go version:       go1.12.5
```

**What happened**
On thanos rule, I click the expr link in the rule tab.

**What you expected to happen**
I expect it go to query tab or query UI to show graph.
But I just get 404.

**How to reproduce it (as minimally and precisely as possible)**:
click links on rule tab

**Full logs to relevant components**

**Anything else we need to know**
thanos running flags
```bash
thanos rule \
--data-dir /var/lib/thanos \
--rule-file /etc/thanos/rules.d/*.yml \
--eval-interval 15s \
--label replica="a" \
--label monitor="rule1" \
--objstore.config-file /etc/thanos/bucket.yml \
--tsdb.retention 8h \
--query my-query:10902 \
--alert.label-drop replica \
--alert.query-url my-query:10902 \
--alertmanagers.url http://my-alert-manager01 \
--alertmanagers.url http://my-alert-manager02 \
--alertmanagers.url http://my-alert-manager03
```

**Comment by jojohappy** [comment]
@Wing924 Hi, we have fixed this bug in #1488 , just behind v0.7.0 only 2 commits, you can try to update thanos rule to the master head.

**Comment by Wing924** [comment]
@jojohappy 
Thank you! I confirmed the master and found it has been fixed.
